troubleshoot/elasticsearch/discovery-troubleshooting.md

```diff
@@ -42,7 +69,7 @@ If the logs suggest that discovery or master elections are failing due to timeou
 
     The threads involved in discovery and cluster membership are mainly `transport_worker` and `cluster_coordination` threads, for which there should never be a long wait. There may also be evidence of long waits for threads in the {{es}} logs, particularly looking at warning logs from `org.elasticsearch.transport.InboundHandler`. See [Networking threading model](elasticsearch://reference/elasticsearch/configuration-reference/networking-settings.md#modules-network-threading-model) for more information.
 
-
+If your cluster has recently lost one or more master-eligible nodes and the logs indicate that no master can be elected, verify that a quorum still exists. A master election requires a majority of the master-eligible nodes to be available (for example, 2 out of 3, or 3 out of 5). If the quorum cannot be met, the cluster will remain unformed until enough nodes are restored. This quorum mechanism is essential for ensuring consistency and preventing split-brain conditions.
```

Comment:
I think this duplicates the information above:

> If the logs or the health report indicate that {{es}} can’t discover enough nodes to form a quorum...

and 

> If the logs or the health report indicate that {{es}} *has* discovered a possible quorum of nodes...

No need for the user to have to work out what a quorum/majority really is, and indeed they often get confused because they need a majority of the master-eligible nodes _that previously made up the cluster_, it's not enough to start some new nodes because those nodes' votes won't yet count in the election. I'd rather we didn't add this paragraph.